extras/embed_data.bzl

```diff
@@ -12,13 +12,56 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""embed_data.bzl provides the go_embed_data rule for embedding data in go files"""
+"""
+  [gazelle rule]: https://github.com/bazelbuild/bazel-gazelle#bazel-rule
```

Comment:
Thanks for moving this in. I think we should still have the header start out with `# Extra rules`, but include a section after `## Contents` called `## Additional resources` that contains these 4 links

Reply:
Sure thing, just updated